handler/status.go

```diff
@@ -73,6 +75,20 @@ func Status(
 				"base64": func(s string) string {
 					return base64.RawURLEncoding.EncodeToString([]byte(s))
 				},
+				"formatHistogram": func(m *dto.Histogram) string {
+					h, fh := histogram.NewModelHistogram(m)
+					if h == nil {
+						return fmt.Sprintf("{ count:%v sum:%v %s}",
+							m.GetSampleCountFloat(),
+							m.GetSampleSum(),
+							histogram.BucketsAsString[float64](histogram.GetAPIFloatBuckets(fh)))
```

Comment:
Wouldn't `return h.String()` do the same?

Reply:
It sure does :+1: